RATapi/run.py

```diff
@@ -19,23 +61,24 @@ def run(project, controls):
     }
 
     horizontal_line = "\u2500" * 107 + "\n"
-
+    display = controls.display != Display.Off
```

Comment:
i think this would be easier to read if the variable name was more 'booley' e.g. `display_on` - if someone's eyes skipped past this line then it'd be easy to expect that `display` is a `Display` object.

alternatively, a bit more work and maybe overkill: if you give the `Display` enum a `__bool__` method you can set the boolean value of each enum value so that `Off` is `False`, and then just directly use `controls.display` throughout this function (which feels consistent with other duck-typing 'falsey' objects, like `Off` feels like it shares a category with `0`, `""`, etc.)

Reply:
I will rename the variable to `display_on` but your __bool__ suggestion could be part of a future refactor